### DISCUSSION TITLE
The desktop icon for the collapsible sidebar in V3 navigation does not switch to the left when the sidebar is collapsed.

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#9429] [Help] [answered]
https://filamentphp.com/docs/3.x/panels/navigation#collapsible-sidebar-on-desktop

https://github.com/filamentphp/filament/assets/121255432/6f500dca-7908-4290-b043-6a5211dcc3d8

**Comment by leandrocfe** [comment, marked as answer]
What is the Filament version?
run `composer update`
